Maya/Exporter/BabylonExporter.GLTFExporter.Material.cs

```diff
@@ -562,47 +562,69 @@ private bool isTextureOk(BabylonTexture texture)
-
-            if (displayPrints)
-            {
-                RaiseVerbose("-----------------------", 3);
-                RaiseVerbose("diffuse=" + diffuse, 3);
-                RaiseVerbose("opacity=" + opacity, 3);
-                RaiseVerbose("specular=" + specular, 3);
-                RaiseVerbose("glossiness=" + glossiness, 3);
-
-                RaiseVerbose("oneMinusSpecularStrength=" + oneMinusSpecularStrength, 3);
-                RaiseVerbose("metallic=" + metallic, 3);
-                RaiseVerbose("diffuseScaleFactor=" + diffuseScaleFactor, 3);
-                RaiseVerbose("baseColorFromDiffuse=" + baseColorFromDiffuse, 3);
-                RaiseVerbose("baseColorFromSpecular=" + baseColorFromSpecular, 3);
-                RaiseVerbose("metallic * metallic=" + metallic * metallic, 3);
-                RaiseVerbose("baseColor=" + baseColor, 3);
-                RaiseVerbose("-----------------------", 3);
-            }
-
-            return new MetallicRoughness
-            {
-                baseColor = baseColor,
-                opacity = opacity,
-                metallic = metallic,
-                roughness = 1 - glossiness
-            };
+        private MetallicRoughness ConvertToMetallicRoughness(SpecularGlossiness specularGlossiness, bool displayPrints = false)
+        {
+            // Hard coded points used to define the specular power to roughness curve.
+            var P0 = new BabylonVector2(0f, 1f);
+            var P1 = new BabylonVector2(0f, 0.1f);
+            var P2 = new BabylonVector2(0f, 0.1f);
+            var P3 = new BabylonVector2(1300f, 0.1f);
+
+            /**
+             * Helper function that defines the bezier curve as well.Given the control points, solve for x based on a given t for a cubic bezier curve
+             * @param t a value between 0 and 1
+             * @param p0 first control point
+             * @param p1 second control point
+             * @param p2 third control point
+             * @param p3 fourth control point
+             * @returns number result of cubic bezier curve at the specified t
+             */
+            float _cubicBezierCurve(float t, float p0, float p1, float p2, float p3)
+            {
+                return
+                (
+                    (1 - t) * (1 - t) * (1 - t) * p0 +
+                    3 * (1 - t) * (1 - t) * t * p1 +
+                    3 * (1 - t) * t * t * p2 +
+                    t * t * t * p3
+                );
+            }
+
+            /*
+             * Helper function that calculates a roughness coefficient given a blinn-phong specular power coefficient
+             * @param specularPower the blinn-phong specular power coefficient
+             * @returns number result of specularPower -> roughness conversion curve.
+             */
+            float _solveForRoughness(float specularPower)
+            {
+                var t = Math.Pow(specularPower / P3.X, 0.333333);

```

Comment:
I can see this is copied from Babylon.js, but I can't remember why there is a power to the 1/3 here. @kcoley Do you remember?

Reply:
If we plot out our converter curve, we get this nice hyperbolic-ish curve:
![shininess to roughness curve](https://user-images.githubusercontent.com/3952729/56536799-cb3bf900-6513-11e9-87ff-f05312312362.png)

Reply:
I'll most likely do a deeper study in whether or not this is correct when I get around to tackling the 3dsMax->Babylon standard material conversion as well.

Hey Kacey, if you still have access to the data that defined this original curve, that would be a huge help.